experiments/AMIP/modular/coupler_driver_modular.jl

```diff
@@ -436,13 +437,19 @@ import_combined_surface_fields!(cs.fields, cs.model_sims, cs.boundary_space, tur
 import_atmos_fields!(cs.fields, cs.model_sims, cs.boundary_space, turbulent_fluxes)
 update_model_sims!(cs.model_sims, cs.fields, turbulent_fluxes)
 
-# 2) each surface component model calculates its own vapor specific humidity (q_sfc)
+# 2) call DSS on each of the simulation states to ensure our initial conditions are continuous fields
```

Comment:
Would it be possible to dss right after initializing the models? The functions above this already create some cache variables that depend on the initialized states. We could also include these as part of the init functions.

Reply:
Yeah, I'll move the dss calls to the init functions